itest/lnd_route_blinding_test.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/btcsuite/btcd/btcec/v2"
 	"github.com/btcsuite/btcd/btcutil"
+	"github.com/go-errors/errors"
```

Comment:
Intentional?

Reply:
nice catch - weird this is the default import path on my IDE, now fixed.